src/DynamoRevitIcons/RevitNodesImages.resx

```diff
@@ -9057,274 +9057,330 @@
   </data>
   <data name="Revit.Elements.Element.AreJoined.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
     <value>
-        iVBORw0KGgoAAAANSUhEUgAAAGEAAABhCAYAAADGBs+jAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8
```

Comment:
this PR seems to replace many of the images for other nodes, is that intended ?

Reply:
Everything seems to be as they should, not sure why it's doing this.